### DISCUSSION TITLE
Migration to GitHub is not complete

### DISCUSSION THREAD

**Discussion by Neustradamus** [windirstat/windirstat#126] [General]
Dear @windirstat team, @assarbad, @NoMoreFood,

Thanks for your work and the new development of this project!

It will be nice to remove old parts like:
- https://sourceforge.net/p/windirstat/code/
- https://sourceforge.net/p/windirstat/wiki/
- https://sourceforge.net/p/windirstat/_list/hg
- https://sourceforge.net/p/windirstat/mailman/
- Clean: https://sourceforge.net/projects/windirstat/files/

Can you add a "move" box like https://sourceforge.net/projects/mumble for example.
- https://sourceforge.net/p/windirstat/

Please update the website with the good link, this GitHub:
- https://sourceforge.net/projects/windirstat/support

To disable/delete tabs, go here: https://sourceforge.net/projects/windirstat/
To add a new tab, go here: https://sourceforge.net/projects/windirstat/
To change informations, go here: https://sourceforge.net/p/windirstat/admin/overview
To export, go here: https://sourceforge.net/p/windirstat/admin/export

The new place can be added in:
- "Moved Project to", here: https://sourceforge.net/p/windirstat/admin/overview
- A new tab, go here: https://sourceforge.net/projects/windirstat/

Thanks in advance.

**Comment by assarbad** [comment]
Why, though?

PS: Also, FYI, I removed your mention of someone who is totally unrelated to the project in your original message (noticed the second one was also unrelated ... also removed).
PPS: I was actually looking into getting the respective repos somehow synchronized in the background, such that the one here is the authoritative one and the others are the backups (also, you seem to have missed [this](https://gitlab.com/windirstat/windirstat) 😉)

**Comment by NoMoreFood** [comment]
I definitely like the idea of syncing the source code for backup purposes.  I also like some of the suggestions of redirecting users to GitHub just so there's no confusion about where we are doing our work or what the active source code is.  I defer to Oliver though since this is more his territory.

**Comment by Neustradamus** [comment]
@assarbad: I have added 4 people from Sourceforge:
- https://sourceforge.net/p/windirstat/_members/

Good catch for Gitlab!

On Sourceforge, you can remove old parts like I have requested, add the "move" box and use SF for files but it must be up-to-date.
Some projects have removed all files, some always use in more "move" box and GitHub main place.

**Comment by Neustradamus** [comment]
Dear @assarbad, @NoMoreFood,

Have you progressed on it?

**Reply by NoMoreFood** [reply]
I have not done anything yet.  We both have our job/family commitments to attend to so give us a little time on this one :-)

**Reply by Neustradamus** [reply]
It is possible to reopen this ticket before the solution?
- https://github.com/windirstat/windirstat/issues/125

**Comment by Neustradamus** [comment]
@NoMoreFood: Have you forgotten all this ticket?

**Reply by NoMoreFood** [reply]
I have not forgotten about it, no.  It's just lower on the priority list and I need to consult with @assarbad on it at some point.

**Comment by NoMoreFood** [comment]
I did most of what you suggested.  We're going to use SourceForge as a backup / download repo.  I setup a GitHub integration so it'll get new releases over there and I'll occasionally do a code push there.

**Comment by Neustradamus** [comment]
@assarbad, @NoMoreFood: We are in 2026, I think that you can remove the old Git tab on Sourceforge:
- https://sourceforge.net/p/windirstat/_list/git
